bin/express-cli.js

```diff
@@ -147,12 +148,32 @@ function createApplication (name, dir) {
     dependencies: {
       'debug': '~2.6.9',
       'express': '~4.16.1'
+    },
+    devDependencies: {
     }
   }
 
+  if (program.typescript) {
+    pkg.scripts['build'] = 'tsc'
+
+    pkg.devDependencies['typescript'] = '~3.7.5'
+    pkg.devDependencies['@types/node'] = '~13.7.0'
```

Comment:
Would this cause an issue when generated and used on an older node.js like, say 10.x?

Reply:
It should be safe on older node environment as the version of the generated codes are controlled via tsconfig.json (line 5, "target": "es5"). Currently it is set compiling codes down to ES5 to support most of the older node.js.

Reply:
Gotcha. I guess i mean typescript-wise. As in I assume that the node 13 types include definitions that do not apply/work on node 10, which then reduces the useful type checking that would prevent that?

Reply:
Yes, you are correct.

As the typing version of node is not one-to-one, perhaps we can adjust it to match the major version of current node.

However, during testing I found that node typing below v8 causes typing mismatch errors so that I suggest setting the minimum of @types/node to v8.

```
// When below v8 (tested with v6.17.1):

node_modules/@types/connect/index.d.ts:21:42 - error TS2689: Cannot extend an interface 'http.IncomingMessage'. Did you mean 'implements'?

21     export class IncomingMessage extends http.IncomingMessage {
```

```js
pkg.devDependencies['@types/node'] = '^' + Math.max(8, parseInt(process.version.replace('v', '')))
// example output => "@types/node": "^12"
```